### ISSUE TITLE
Support Window-Dragging from Bookmarks Toolbar

### ISSUE THREAD

**Issue by jonathansampson** [brave/browser-laptop#9384]
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
Users cannot drag the window around if they happen to initiate the drag event from unused space on the bookmarks toolbar. This is particularly annoying since the user could move their mouse up a few pixels, next to the address bar, and have a successful experience.

- Platform (Win7, 8, 10? macOS? Linux distro?):
Desktop

- Brave Version (revision SHA):
0.16.9

- Steps to reproduce:
    1. Ensure that your Bookmarks Toolbar is visible
    2. Attempt to click-drag window from empty toolbar space
    
- Actual result:
Window does not drag.

- Expected result:
Window drags.

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Yes

- Can this issue be consistently reproduced?
Yes

**Comment by jonathansampson** [comment]
You can test the effect by pressing Shift + F8, and running the following in your console:

    document.querySelector('div.bookmarksToolbar').style['-webkit-app-region'] = 'drag'

**Comment by bsclifton** [comment]
@jonathansampson adding that style isn't (by itself) a suitable fix.  If you try adding it, you'll notice you can no longer right click the bookmarks toolbar (well... you can, but it will only show the default Windows caption menu... not Brave's context menu)

The fix for this would be:
- Add the draggable style
- Try to intercept the right click in React (for the bookmarks toolbar component)

**Comment by bsclifton** [comment]
(Also, marked as `OS/Windows` and `OS/linux-distros` since this works fine on macOS)

**Comment by luixxiul** [comment]
I tried to find a solution by setting margin/padding but ^^ is the proper one I think.

**Comment by bsclifton** [comment]
Per my above comment, the right clicking => context menu issue would need to be solved with https://github.com/brave/browser-laptop/issues/4244

**Comment by bsclifton** [comment]
Blocked by https://github.com/brave/browser-laptop/issues/4244

**Comment by bsclifton** [comment]
Closing as `wontfix` since we won't be able to address https://github.com/brave/browser-laptop/issues/4244, which blocks this.  This is resolved in [brave-core](https://github.com/brave/brave-browser)
